background/redux-slices/utils/0x-swap-utils.ts

```diff
@@ -63,11 +63,11 @@ export async function getAssetPricePoint(
       return contractAddress
     })
 
-  const unitPricePoint = Object.values(
+  const [unitPricePoint] = Object.values(
     await getTokenPrices(assetPricesNetworks, USD, network)
-  )[0]
+  )
 
-  return getPricePoint(asset, unitPricePoint)
+  return (unitPricePoint && getPricePoint(asset, unitPricePoint)) || undefined
```

Comment:
Thoughts on being a little more explicit here?

```suggestion
  return unitPricePoint === undefined
    ? undefined
    : getPricePoint(asset, unitPricePoint))
```

Reply:
Thoughts are positive!